src/components/Capture/index.js

```diff
@@ -47,19 +50,20 @@ export default class Capture extends Component {
 
     const { actions, socket, documentType } = this.props
     const methods = {
-      'document': (payload) => {
-        payload.isValid = false
-        payload.documentType = documentType
-        socket.sendMessage(JSON.stringify(payload))
-        actions.documentCapture(payload)
+      'document': (methodPayload) => {
+        var data = methodPayload.data;
```

Comment:
any reason why we are going with no semicolon? there are some cases where no semicolon can cause issues.